src/xmipp/libraries/reconstruction/resolution_monogenic_signal.cpp

```diff
@@ -32,558 +32,306 @@ void ProgMonogenicSignalRes::readParams()
-	addParamsLine("  [--minRes <s=30>]         : Minimum resolution (A)");
-	addParamsLine("  [--maxRes <s=1>]          : Maximum resolution (A)");
-	addParamsLine("  [--trimmed <s=0.5>]       : Trimming percentile");
-	addParamsLine("  [--exact]                 : The search for resolution will be exact (slower) of approximated (fast).");
-	addParamsLine("                            : Usually there are no difference between both in the resolution map.");
-	addParamsLine("  [--noiseonlyinhalves]     : The noise estimation is only performed inside the mask");
-	addParamsLine("  [--filtered_volume <vol_file=\"\">]       : The input volume is locally filtered at local resolutions.");
-	addParamsLine("  [--significance <s=0.95>]    : The level of confidence for the hypothesis test.");
-	addParamsLine("  [--md_outputdata <file=\".\">]  : It is a control file. The provided mask can contain voxels of noise.");
-	addParamsLine("                                  : Moreover, voxels inside the mask cannot be measured due to an unsignificant");
-	addParamsLine("                                  : SNR. Thus, a new mask is created. This metadata file, shows, the number of");
-	addParamsLine("                                  : voxels of the original mask, and the created mask");
-	addParamsLine("  [--automatic]                   : Resolution range is not neccesary provided");
-	addParamsLine("  [--threads <s=4>]               : Number of threads");
+	addUsageLine("MONORES: This algorithm estimate the local resolution map from a single reconstruction");
+	addUsageLine("or two half maps.");
+	addUsageLine("Reference: J.L. Vilas et al, MonoRes: Automatic and Accurate Estimation of ");
+	addUsageLine("Local Resolution for Electron Microscopy Maps, Structure, 26, 337-344, (2018).");
+	addUsageLine("  ");
+	addUsageLine("  ");
+	addParamsLine("  --vol <vol_file=\"\">         : Input map to estimate its local resolution map.");
+	addParamsLine("                                : If two half maps are used, it will be the first half map.");
+	addParamsLine("  --minRes <s=30>               : Lowest resolution in (A) for the resolution range");
+	addParamsLine("                                : to be analyzed.");
+	addParamsLine("  --maxRes <s=1>                : Highest resolution in (A) for the resolution range");
+	addParamsLine("                                : to be analyzed.");
+	addParamsLine("  --sampling_rate <s=1>         : Sampling rate (A/px)");
+	addParamsLine("  -o <output_file=\"\">         : Folder where the results will be stored.");
+	addParamsLine("                                : If two half maps are used this should be the first one.");
+	addParamsLine("  [--vol2 <vol_file=\"\">]      : (Optional but recommended) Second half map to estimate its");
+	addParamsLine("                                : local resolution map. The first one will be the --vol label.");
```

Comment:
So is this some reference volume? Why should it be otherwise recommended to process two volumes at the same time?

Reply:
Yes, a kind of. They are the two half maps. But the algorithm work also with a single map. That's the reason of using vol and vol2 instead of half1 and half2.